### DISCUSSION TITLE
Neue Installation auf Home-Assistant Intel NUC & Übernahme aller Geräte von defekter CCU2

### DISCUSSION THREAD

**Discussion by janningnetworks** [OpenCCU/OpenCCU#1689] [🇩🇪 Fragen&Antworten] [answered]
Hallo,
ich habe eine defekte CCU2 (ist leider mit dem letzten Firmware-Update gecrashed) 
komme immer wieder in dieses Szenario:
https://homematic-forum.de/forum/viewtopic.php?f=37&t=53199&p=529926&hilit=cux+segmentation#p529926 
Neustart über 
`/bin/ReGaHss.normal` -f /etc/rega.conf -l 2 -c `
läuft in einen SegFault

Jetzt möchte ich gerne komplett neu beginnen und alle Geräte in eine neue RaspberryMatic Installation importieren. Geht das irgendwie? Ich habe ein Backup von August 2020 - da ist auch an den Einstellungen oder Geräten nix mehr geändert worden. Auf die alte CCU2 komme ich per SSH drauf. Aber WebGui geht halt nicht mehr.

Idealerweise würde ich das dann auf meinen HomeAssistant Intel NUC packen - da gibt es ja wohl ein Addon.
Ich benötige dann nur noch einen USB-Funkadapter, richtig?

Vielen Dank!

**Comment by jens-maus** [comment, marked as answer]
> Idealerweise würde ich das dann auf meinen HomeAssistant Intel NUC packen - da gibt es ja wohl ein Addon. Ich benötige dann nur noch einen USB-Funkadapter, richtig?

Genau so ist es. Besorg dir ein `HmIP-RFUSB` als Funkmodul, steck es via USB an die NUC. Dann installier das RaspberryMatic Add-on innerhalb von HomeAssistant und dann solltest du dein lauffähiges CCU2 Backup ganz normal dort einspielen/nutzen können.
